### PR TITLE
Pull Request Detail GitHub Enterprise Avatars 

### PR DESCRIPTION
**Pull request by StanleyGoldman**
Related to #416

This pull request merged the work from #1121 into #1096 and builds upon it
This corrects the display of user avatars on GitHub Enterprise in a Pull Request Detail

Depends on
- [ ] #1121 getting merged to master
- [ ] Needs additional functionality to detect and change the displayed here
https://github.com/Microsoft/vscode-pull-request-github/blob/1fa13ebb1bc50e946c27d55aec12c6ac90add710/preview-src/user.tsx#L5-L8

### CODE REVIEW

**Comment by StanleyGoldman** [comment]
image

File: preview-src/index.css [hunk 1]
```diff
@@ -82,13 +82,18 @@ body .comment-container .avatar-container a {
...
 
-body .comment-container .avatar-container img.avatar {
+body .comment-container .avatar-container img.avatar,
+body .comment-container .avatar-container .avatar-icon svg {
 	margin-right: 0;
 }
 
-body img.avatar {
+body img.avatar,
+body span.avatar-icon svg {
```
Comment (queerviolet) [inline preview-src/index.css]: Did we decide to use a conditional `filter: invert(100%);` rule here, to get the default avatar icon to align with the theme?

**Comment by queerviolet** [comment]
I can make that change, actually. Thanks for DI'ing the change :)

File: src/github/githubRepository.ts [hunk 2]
```diff
@@ -206,7 +209,29 @@ export class GitHubRepository implements IGitHubRepository, vscode.Disposable {
...
+				if(err && err instanceof Error) {
+					response = (<any> err).response as AxiosResponse;
+				}
+			}
+
+			if(response && response.status === 200) {
+				this._repositoryReturnsAvatar = true;
+			}
+
+			this._repositoryReturnsAvatar = false;
```
Comment (RMacfarlane) [inline src/github/githubRepository.ts]: I think we're missing an `else` here, or a return in the 200 check

Comment (StanleyGoldman) [reply src/github/githubRepository.ts]: Thanks, I'll send a PR to fix this.

Comment (StanleyGoldman) [reply src/github/githubRepository.ts]: https://github.com/Microsoft/vscode-pull-request-github/pull/1145

**Pull request merged**